crates/wdl-engine/src/cache/hash.rs

```diff
@@ -64,6 +65,8 @@ enum ValueKind {
     Input,
     /// The value is an `Output` (hidden type).
     Output,
+    /// The value is an `EnumVariant`.
+    EnumVariant,
```

Comment:
It's important that this is last so that its addition doesn't bust previously built caches.